wiki/en/Software-Manual.md

```diff
@@ -224,7 +224,7 @@ If the Auto setting is enabled, the jitter buffers
 of your local Client and the remote server are set automatically based on measurements of the network and sound card
 timing jitter. If the Auto check is enabled, the jitter buffer size faders are disabled (they cannot be moved with the mouse).
 
-### Enable small network buffers
+### Small Network Buffers
```

Comment:
This could break some links, but should also show up if we run the linkchecker.

Reply:
`$ grep -r 'Enable small network buffers' wiki/en* assets/ *html *xml` comes back with nothing, at least.